src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,9 @@
 package core.basesyntax;
 
 public class MainApp {
-
+Machine[] machines = new Machine[] {new Truck(), new Bulldozer(), new Excavator()};
+for (int i = 0;i < machine.length;i++) {
```

Comment:
There is a typo in the array name. It should be `machines` instead of `machine` to match the declared array name.